opensrp-chw/build.gradle

```diff
@@ -220,4 +220,4 @@ tasks.withType(Test) {
 
 coveralls {
     jacocoReportPath = "${buildDir}/reports/jacoco/jacocoRootReport/merged.xml"
-}
+}
```

Comment:
Whats this change?

Reply:
Probably must have happened during the merge